tests/config-override.test.sh

```
@@ -208,6 +208,68 @@ warn_count=$(printf "%s\n" "$_LAST_STDERR" | grep -cE "fanout_concurrency.resear
+_isolate '
+  cat > .claude/uberdev.local.md <<EOF
+---
+uberthink:
+  loop_back_cap: 0
+---
+EOF
+  out="$(uberdev_read_int_in_range uberthink.loop_back_cap UBERDEV_UBERTHINK_LOOP_BACK_CAP 0 10 3)"
+  echo "U2I_RESULT=[$out]" >&2
+'
+grep -qF "U2I_RESULT=[0]" <<<"$_LAST_STDERR" \
+  && pass "U2i: min=0 key configured 0 -> 0 (not the default 3)" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)